src/interactions/interaction.ts

```diff
@@ -11,8 +11,53 @@ module Plottable {
      */
     protected _componentToListenTo: Component;
```

Comment:
Not suggestive enough without prior knowledge of how this class works. maybe `_componentAttachedTo` would work (doing this change)